ranger/data/scope.sh

```diff
@@ -77,13 +76,39 @@ handle_extension() {
             odt2txt "${FILE_PATH}" && exit 5
             exit 1;;
 
+	# RTF and DOC
+	rtf|doc)
+	    # Preview as text conversion
+	    # note: catdoc does not always work for .doc files
+	    catdoc "${FILE_PATH}" && exit 5
+	    exit 1;;
+
+	# DOCX
+	docx)
+	    # Preview as markdown conversion
+	    pandoc -t markdown "${FILE_PATH}" && exit 5
+	    exit 1;;
+
+	# XLSX
+	xlsx)
```

Comment:
Why not xlsx|xls?

Reply:
Because they use different programs (xls2csv and xlsx2csv). The 2 types of files are very different.

Reply:
I just added some info on how to get those applications. It took me some time to look for and test applications which work for those conversions.

Reply:
You're right, I hadn't noticed the extra `x`.

Btw, does scope already handle .odt and the other openoffice/libreoffice formats?

Reply:
It does accept odt|ods|odp|sxw

But none of the formats that I added.